dapp/src/components/TransactionModal/ActiveStakingStep/DepositBTCModal.tsx

```diff
@@ -53,7 +53,10 @@ export default function DepositBTCModal() {
 
   const onDepositBTCError = useCallback(
     (error: unknown) => {
-      if (eip1193.didUserRejectRequest(error)) {
+      if (
+        eip1193.didUserRejectRequest(error) ||
+        ledgerLive.didUserRejectRequest(error)
```

Comment:
Can we combine these two with one common utils function?

Reply:
8b364a0